File: src/main/java/com/keyboardsamurais/apps/config/EnvUtils.java
package com.keyboardsamurais.apps.config;

import io.github.cdimascio.dotenv.Dotenv;

public class EnvUtils {
    public static Dotenv dotenv;

    public static String getEnv(String key) {
        if (dotenv == null) {
            dotenv = Dotenv.configure().ignoreIfMissing().ignoreIfMalformed().load();
        }
        return dotenv.get(key);
    }
}

File: src/main/java/com/keyboardsamurais/apps/config/EnvUtils.kt
package com.keyboardsamurais.apps.config

import io.github.cdimascio.dotenv.dotenv

object EnvUtils {
    private val dotenv = dotenv {
        ignoreIfMalformed = true
        ignoreIfMissing = true
    }

    fun getEnv(key: String): String {
        return dotenv[key]
    }
}
